master/buildbot/steps/vstudio.py

```diff
@@ -386,6 +383,13 @@ class VC12(VC11):
 VS2013 = VC12
 
 
+# VC14 doesn't look like it needs extra stuff.
+class VC14(VC12):
+    default_installdir = 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0'
```

Comment:
I understand that this follows the way in other classes, however I'd suggest to use `r'...\...'` to improve readability.